phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -942,6 +942,9 @@ private boolean addIndexesFromParentTable(MetaDataMutationResult result, Long re
                 } else {
                     indexesToAdd.add(PTableImpl.builderWithColumns(index, getColumnsToClone(index))
                             .setTableName(modifiedIndexName)
+                            .setParentName(view.getName())
```

Comment:
This is a bug on the existing code where the parent table name is not set properly when the index name are generated  for views. My code triggers this bug when the code attempts to get index maintainers for a given view. @twdsilva suggested this would be the fix and it fixed the issue.